File: 412-fizz-buzz/fizz-buzz.java
class Solution 
{
    public List<String> fizzBuzz(int n) 
    {
        List<String> str=new ArrayList<>();
        for(int x=1;x<=n;x++)
        {
            if(x%3==0 && x%5==0)
            {
                str.add("FizzBuzz");
            }
            else if(x%3==0)
            {
                str.add("Fizz");
            }
            else if(x%5==0)
            {
                str.add("Buzz");
            }
            else
            {
                str.add(Integer.toString(x));
            }
        }
        return str;
    }
}

File: 412-fizz-buzz/fizz-buzz.cpp
class Solution 
{
public:
    vector<string> fizzBuzz(int n) 
    {
        vector<string> str;
        for(int x=1;x<=n;x++)
        {
            if(x%3==0 && x%5==0)
            {
                str.push_back("FizzBuzz");
            }
            else if(x%3==0 )
            {
                str.push_back("Fizz");
            }
            else if(x%5==0 )
            {
                str.push_back("Buzz");
            }
            else
            {
                str.push_back(to_string(x));
            }
        }
        return str;
    }
};